seatunnel-api/src/main/java/org/apache/seatunnel/api/table/converter/BasicTypeDefine.java

```diff
@@ -31,6 +31,7 @@ public class BasicTypeDefine<T> implements Serializable {
     protected String columnType;
     // e.g. `varchar` for MySQL
     protected String dataType;
+    protected int sqlType;
```

Comment:
I think we can use nativeType instead of sqlType.

Reply:
> I think we can use nativeType instead of sqlType.

no,  like 'INT' cannot be converted to standard JDBCType, SQLType doesn't need to worry about this issue

Reply:
OK, please add some comment to tell devs the sqlType come from jdbc, not come from seatunnel sql type. cc @hailin0